cmd/wallet-web/src/store/modules/options.js

```diff
@@ -319,5 +319,13 @@ export default {
     getStaticAssetsUrl(state) {
       return state.agentOpts['staticAssetsUrl'];
     },
+    async getStaticAssetsConfigData(state) {
+      let staticUrl = state.agentOpts['staticAssetsUrl'];
+      if (staticUrl) {
+        const response = await axios.get(`${staticUrl}/config/credentialDisplayData.json`);
+        return response.data;
+      }
+      return false;
```

Comment:
why it returns object sometimes and bool otherwise?

Reply:
updated